<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>940</width>
    <height>705</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridWidget" native="true">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>491</width>
      <height>441</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 255, 127);</string>
    </property>
    <layout class="QGridLayout" name="gridLayout"/>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>80</y>
      <width>271</width>
      <height>361</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridOptions">
     <item row="4" column="0">
      <widget class="QPushButton" name="newGame">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">border-color: rgb(255, 0, 4);
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
font: 9pt &quot;Algerian&quot;;
selection-background-color: qradialgradient(spread:repeat, cx:0.5, cy:0.5, radius:0.077, fx:0.5, fy:0.5, stop:0 rgba(0, 169, 255, 147), stop:0.497326 rgba(0, 0, 0, 147), stop:1 rgba(0, 169, 255, 147));</string>
       </property>
       <property name="text">
        <string>NEW GAME</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="listButtonHere">
       <property name="font">
        <font>
         <family>SimSun</family>
         <pointsize>14</pointsize>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;SimSun&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelRed">
       <property name="font">
        <font>
         <family>Kristen ITC</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="labelBlack">
       <property name="font">
        <font>
         <family>Kristen ITC</family>
         <pointsize>12</pointsize>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>529</y>
      <width>491</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="movesLayout">
     <item>
      <widget class="QPushButton" name="play">
       <property name="styleSheet">
        <string notr="true">color:rgb(255, 170, 0);
background-color: rgb(85, 85, 0);
font: 14pt &quot;Viner Hand ITC&quot;;</string>
       </property>
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="labelMessage">
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(85, 85, 0);
font: 700 italic 14pt &quot;Sitka Display&quot;;</string>
         </property>
         <property name="text">
          <string>msg curent player</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelWinner">
         <property name="styleSheet">
          <string notr="true">font: italic 12pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 0);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>940</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
